### ISSUE TITLE
Cannot start Redis 7.0 with Sentinel

### ISSUE THREAD

**Issue by vytas7** [jd/pifpaf#173]
It seems that `pifpaf` cannot start Redis 7.0 with Sentinel. Installing `redis-server` & `redis-sentinel` on Ubuntu 24.04 beta, and trying to run `pifpaf` results in:
```
(venv) root@316876421b0b:/tmp# pifpaf run redis --sentinel --
WARNING [pifpaf.util] `psutil.Popen(pid=3289, name='redis-sentinel', status='zombie', started='15:28:06')` is already gone, sending SIGKILL to its process group
ERROR [pifpaf] Program did not print: `# Sentinel (runid|ID) is'
Output: b"3289:X 18 Apr 2024 15:28:07.113 # Failed to resolve hostname 'localhost'\n\n*** FATAL CONFIG FILE ERROR (Redis 7.0.15) ***\nReading the configuration file, at line 3\n>>> 'sentinel monitor pifpaf localhost 6379 1'\nCan't resolve instance hostname.\n"
```

My colleague found out that it might be enough to add the following configuration line:
```
sentinel resolve-hostnames yes
```
Or alternatively, one might be able to simply use `127.0.0.1` instead of `localhost` on all Redis versions.

Normal Redis (without Sentinel) works.

**Comment by kajinamit** [comment]
Because Ubuntu Jammy ships redis 6.0, it's probably better to use ipaddress instead. It's strange that we haven't caught this problem although we run some CI Jobs using this feature.

**Comment by tobias-urdin** [comment]
released in 3.2.1

**Comment by vytas7** [comment]
Thank you @kajinamit and @tobias-urdin! 💯
